sync/class.jetpack-sync-actions.php

```diff
@@ -178,6 +180,10 @@ static function do_full_sync( $modules = null ) {
 		self::do_cron_sync(); // immediately run a cron sync, which sends pending data
 	}
 
+	static function schedule_full_sync_constants_and_callables() {
```

Comment:
this function probably needs a different name, since it's actually running the full sync, not scheduling it